Source/Core/Core/ConfigManager.h

```diff
@@ -8,6 +8,7 @@
 #include <string>
 #include <vector>
 
+#include "Common/File.h"
```

Comment:
This makes me sad. (ConfigManager is included by tons of things, so this increases the number of files that need to be recompiled when File.h is edited)